src/actions/upload.js

```diff
@@ -81,6 +80,7 @@ export function uploadFile(file, options = {}) {
             const {enableSignedUrls} = options
             const body = {
               playback_policy: [enableSignedUrls ? 'signed' : 'public'],
+              // eslint-disable-next-line no-warning-comments
```

Comment:
is this `eslint-disalbe-next-line` needed?

Reply:
You're right, might be better to remove this. 

Removed in a43938a